packages/react/package.json

```diff
@@ -5,17 +5,16 @@
   "author": "cloudinary",
   "license": "MIT",
   "repository": "https://github.com/cloudinary/frontend-frameworks",
-  "main": "dist/index.js",
-  "umd:main": "./dist/index.umd.js",
-  "module": "dist/index.esm.js",
+  "main": "index.umd.js",
```

Comment:
cool. 

The main field makes sure that Node users using require will be served the UMD version. The module field is not an official npm feature but a common convention among bundlers to designate how to import an ESM version of our library. I believe this is also only relevant when using rollup

Reply:
Exactly so,
Module is relevant for webpack as well